bugbug/similarity.py

```diff
@@ -626,6 +628,65 @@ def get_distance(self, query1, query2):
         raise NotImplementedError
 
 
+class ElasticSearchSimilarity(BaseSimilarity):
+    def __init__(self, cleanup_urls=True, nltk_tokenizer=False):
+        super().__init__(cleanup_urls=cleanup_urls, nltk_tokenizer=nltk_tokenizer)
+        self.elastic_search = Elasticsearch()
+        assert (
+            self.elastic_search.ping()
+        ), "Check if Elastic Search Server is running by visiting http://localhost:9200"
+
+    def get_text(self, bug):
+        return "{} {}".format(
+            bug["summary"], " ".join([comment["text"] for comment in bug["comments"]])
+        )
+
+    def text_preprocess(self, text):
```

Comment:
Just use the `text_preprocess` from the base class (if you want to avoid stemming, make it optional like lemmatization).